spec/models/reservation_spec.rb

```diff
@@ -420,6 +421,85 @@
     end
   end
 
+  context 'with a reservation starting the next day that passes length limit' do
+    subject(:reservation) do
+      r = FactoryGirl.build(:valid_reservation)
+      r.equipment_model.max_checkout_length = 3
+      r.equipment_model.save
+      r
+    end
+    let!(:consecutive) do
+      FactoryGirl.create(:reservation,
```

Comment:
Doesn't `create` actually save the record? I think you could just use `build` instead, or am I mis-remembering FactoryGirl?